### DISCUSSION TITLE
Migrating from Slack to Discord

### DISCUSSION THREAD

**Discussion by ConorOkus** [lightningdevkit/rust-lightning#1580] [Announcements]
In the last dev [meeting](https://docs.google.com/document/d/1VqpkVQLCIA__pwMWzmzcOb015ZmGvVJabOOYBjsVJbQ/edit) we decided to fully migrate from Slack to Discord in order to unify the community under one chat platform. We set up the Discord in early 2022, kept the Slack open and let people choose their preferred platform. However, the Slack pricing model makes it unsuitable for building a large FOSS public community.  

Discord can bring many necessary features and benefits to our community including:

- We get to preserve our conversation history and hold on to important threads
- Superior video sharing and voice channels for fun and interesting activities
- Can schedule events in the platform for members to engage with and grow as LDK developers
- Advanced moderation tools to keep our community safe and accessible for all

Moving forward members in the community should aim to exclusively post new stuff to Discord, while the Slack dies down over the coming weeks/months before officially closing.

We believe semi-anonymous participation is an important feature for whatever chat platform we use. A few of the community access Discord via a Matrix bridge that seems to work well. You can join that [here](https://matrix.to/#/#ldk-discord:bitcoin.ninja)

Click this [invite](https://discord.gg/5AcknnMfBw) to join the Discord. 

Please leave any comments or questions here if you have concerns.

**Comment by rndhouse** [comment]
@ConorOkus did you guys consider [Matrix](https://matrix.org/)? I would be interested in your evaluation.

**Reply by TheBlueMatt** [reply]
There's a bridge at #ldk-discord:bitcoin.ninja, though ultimately the % of folks who want to join who have Discord already is much higher than those with Matrix.

**Reply by rndhouse** [reply]
Ah I figured it might come down to which platform was most popular. Good to know there's a bridge!
